src/script/ui/component/form/form.jsx

```diff
@@ -114,7 +123,7 @@ function Field(props) {
   return (
     <Label
       className={classNames({ 'data-error': dataError })}
-      error={dataError}
+      error={dataError ? 1 : 0}
```

Comment:
I see that dataError can belong to Object type in some cases. Do you  really need to convert it to Boolean?